aries-site/src/examples/templates/forms/Shipping/ContactInfomation.js

```diff
@@ -0,0 +1,38 @@
+import React from 'react';
+import { Box, FormField, MaskedInput, Text, TextInput } from 'grommet';
+import { emailMask, emailValidation, phoneMask } from '../utils/FormValidation';
+
+export const ContactInformation = () => (
```

Comment:
I was thinking these would be defined and exported within the ShippingExample file, but I think this is fine.

Reply:
I can put back but I did include the file in the code preview

Reply:
I think it is fine to leave as is.